### PR TITLE
Refactor several imperfections

### PR DESCRIPTION
**Pull request by lootag**

### CODE REVIEW

**Review by celaus — APPROVED** [review]

File: go/actions-client.go [hunk 1]
```diff
@@ -79,8 +82,8 @@ func (a ActionsClient) GetAction(ctx context.Context, req *GetActionRequest) (*a
 	return client.GetAction(callProperties.Ctx, req, callProperties.CallOptions...)
 }
 
-// UpdateAction updates an action.
-func (a ActionsClient) UpdateAction(ctx context.Context, req *ReplaceActionRequest) (*actions.ReplaceActionResponse, error) {
+// Update updates an action.
+func (a ActionsClient) Update(ctx context.Context, req *ReplaceActionRequest) (*actions.ReplaceActionResponse, error) {
```
Comment (celaus) [inline go/actions-client.go:86]: this should be replace

File: go/alerts-client.go [hunk 2]
```diff
@@ -161,8 +167,8 @@ func (a AlertsClient) GetAlert(ctx context.Context, req *alerts.GetAlertDefReque
 	return client.GetAlertDef(callProperties.Ctx, req, callProperties.CallOptions...)
 }
 
-// UpdateAlert updates an alert.
-func (a AlertsClient) UpdateAlert(ctx context.Context, req *alerts.ReplaceAlertDefRequest) (*alerts.ReplaceAlertDefResponse, error) {
+// Update updates an alert.
+func (a AlertsClient) Update(ctx context.Context, req *alerts.ReplaceAlertDefRequest) (*alerts.ReplaceAlertDefResponse, error) {
```
Comment (celaus) [inline go/alerts-client.go:171]: Replace

File: go/dashboard-folders-client.go [hunk 3]
```diff
@@ -53,8 +53,8 @@ func (c DashboardsFoldersClient) GetDashboardsFolders(ctx context.Context, req *
 	return client.ListDashboardFolders(callProperties.Ctx, req, callProperties.CallOptions...)
 }
 
-// UpdateDashboardsFolder updates a dashboard folder.
-func (c DashboardsFoldersClient) UpdateDashboardsFolder(ctx context.Context, req *dashboards.ReplaceDashboardFolderRequest) (*dashboards.ReplaceDashboardFolderResponse, error) {
+// Update updates a dashboard folder.
+func (c DashboardsFoldersClient) Update(ctx context.Context, req *dashboards.ReplaceDashboardFolderRequest) (*dashboards.ReplaceDashboardFolderResponse, error) {
```
Comment (celaus) [inline go/dashboard-folders-client.go:57]: Replace

File: go/dashboards-client.go [hunk 4]
```diff
@@ -74,8 +74,8 @@ func (d DashboardsClient) GetDashboard(ctx context.Context, req *GetDashboardReq
 	return client.GetDashboard(callProperties.Ctx, req, callProperties.CallOptions...)
 }
 
-// UpdateDashboard updates a dashboard.
-func (d DashboardsClient) UpdateDashboard(ctx context.Context, req *ReplaceDashboardRequest) (*dashboards.ReplaceDashboardResponse, error) {
+// Update updates a dashboard.
```
Comment (celaus) [inline go/dashboards-client.go:77]: Replace

File: go/events2metrics-client.go [hunk 5]
```diff
@@ -53,8 +53,8 @@ func (e Events2MetricsClient) GetEvents2Metric(ctx context.Context, req *e2m.Get
 	return client.GetE2M(callProperties.Ctx, req, callProperties.CallOptions...)
 }
 
-// UpdateEvents2Metric updates a metric.
-func (e Events2MetricsClient) UpdateEvents2Metric(ctx context.Context, req *e2m.ReplaceE2MRequest) (*e2m.ReplaceE2MResponse, error) {
+// Update updates a metric.
+func (e Events2MetricsClient) Update(ctx context.Context, req *e2m.ReplaceE2MRequest) (*e2m.ReplaceE2MResponse, error) {
```
Comment (celaus) [inline go/events2metrics-client.go:57]: Replace

File: go/slis-client.go [hunk 6]
```diff
@@ -11,8 +11,8 @@ type SLIClient struct {
 	callPropertiesCreator *CallPropertiesCreator
```
Comment (celaus) [inline go/slis-client.go:11]: this entire file  can be deleted :)

**Review by celaus — APPROVED** [review]

File: go/actions-client.go (hunk 1 above)
Comment (lootag) [reply go/actions-client.go]: Done :-)

File: go/alerts-client.go (hunk 2 above)
Comment (lootag) [reply go/alerts-client.go]: Done

File: go/dashboard-folders-client.go (hunk 3 above)
Comment (lootag) [reply go/dashboard-folders-client.go]: Done

File: go/dashboards-client.go (hunk 4 above)
Comment (lootag) [reply go/dashboards-client.go]: Done

Comment (lootag) [reply go/dashboards-client.go]: Done

File: go/events2metrics-client.go (hunk 5 above)
Comment (lootag) [reply go/events2metrics-client.go]: Done

File: go/slis-client.go (hunk 6 above)
Comment (lootag) [reply go/slis-client.go]: Done

**Pull request merged**